webui/react/src/components/LogViewer.tsx

```diff
@@ -102,6 +102,7 @@ const LogViewer: React.FC<Props> = forwardRef((
   { onScrollToTop, ...props }: Props,
   ref?: React.Ref<LogViewerHandles>,
 ) => {
+  const onDownload = props.onDownload;
```

Comment:
**suggestion:** move this into `props` deconstruction (before `onScrollToTop`)?

Also can you move the `onDownload` interface declaration to be alphabetical order up top? 🙏